src/plugins/janus_audiobridge.c

```diff
@@ -4900,7 +4902,7 @@ static json_t *janus_audiobridge_process_synchronous_request(janus_audiobridge_s
 			g_snprintf(error_cause, 512, "Can't reset (not in a room)");
 			goto prepare_response;
 		}
-		participant->reset = TRUE;
+		g_atomic_int_set(&participant->reset, TRUE);
```

Comment:
For atomic ints, we use numbers and not boolean values, so better have `1` there. No actual change in functionality, just code style.